content/blog/automate-boot-from-san-target-configurations-using-redfish.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Automate boot from SAN target configurations using Redfish
+date: 2023-06-01T20:13:38.207Z
+featuredBlog: false
+author: HPE Global Sales Engineering (GSE) team
+authorimage: /img/Avatar5.svg
+disable: false
+tags:
```

Comment:
Please add a tag:
ilo-restful-api

This will allow your blog post to be tied to the landing page "iLO RESTful API": https://developer.hpe.com/platform/ilo-restful-api/home/